spec/models/portfolio_item_spec.rb

```diff
@@ -1,5 +1,6 @@
 describe PortfolioItem do
-  let(:item) { build(:portfolio_item) }
+  let(:tenant) { create(:tenant) }
+  let(:item) { PortfolioItem.new(:tenant_id => tenant.id, :name => "test", :display_name => "display test") }
```

Comment:
What's wrong with just `build(:portfolio_item)`? The factory already includes the tenant and has a sequence for both the name and display name.

Reply:
Interesting. That failed for me yesterday. But possibly it was due to other changes I was looking at earlier.  Either way. Fixed.